Dockerfile.dev-env

```diff
@@ -16,6 +16,9 @@ RUN --mount=type=cache,target=/go/pkg/mod \
   && GOBIN=/workspace/bin go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest \
   && cp $(/workspace/bin/setup-envtest use ${ENVTEST_K8S_VERSION} -p path)/* /usr/local/bin
 
+# Get AdminPluginDefinitions from https://github.com/cloudoperators/greenhouse-extensions
```

Comment:
I guess we talked to much about AdminPlugins these last days. 😃 
```suggestion
# Get PluginDefinitions from https://github.com/cloudoperators/greenhouse-extensions
```

Reply:
Actually, I wanted to convey that we _only_ get the `PluginDefinitions` relevant for running Greenhouse core... As for now only `ServiceProxy`

Reply:
I agree with the argument, but this wording is only obvious for anyone being part of the discussion. Others might not see what we consider an adminplugin.
Maybe this could be added to line 35 e.g. "Copy over PluginDefinitions required to run Greenhouse core"